includes/shortcodes/class-ssp-shortcode-podcast_playlist.php

```diff
@@ -33,17 +33,18 @@ function shortcode( $params ) {
 
 		// Parse shortcode attributes
 		$atts = shortcode_atts( array(
-			'type'		=> 'audio',
-			'series'	=> '',
-			'order'		=> 'ASC',
-			'orderby'	=> 'menu_order ID',
-			'include'	=> '',
-			'exclude'   => '',
-			'style'		=> 'light',
-			'tracklist' => true,
-			'tracknumbers' => true,
-			'images'	=> true,
-            'limit'     => -1
+			'type'		    => 'audio',
+			'series'	    => '',
+			'order'		    => 'ASC',
+			'orderby'	    => 'menu_order ID',
+			'include'	    => '',
+			'exclude'       => '',
+			'style'		    => 'light',
+			'player_style'  => 'mini',
```

Comment:
New `player_style` attribute, not in use as we are forcing the mini player, but it will soon be in use.